src/Raven.Server/Documents/ETL/Providers/Raven/RavenEtlItem.cs

```diff
@@ -14,10 +18,22 @@ public RavenEtlItem(Document document, string collection) : base(document, colle
 
         public RavenEtlItem(Tombstone tombstone, string collection, EtlItemType type) : base(tombstone, collection, type)
         {
-            if (tombstone.Type == Tombstone.TombstoneType.Attachment)
-            {
-                AttachmentTombstoneId = tombstone.LowerId;
-            }
+        }
+
+        public RavenEtlItem(DocumentsOperationContext context, AttachmentTombstoneReplicationItem attachment)
+        {
+            DocumentId = context.GetLazyString(attachment.Key.ToString());
+
+            //AttachmentTombstoneId = attachment.Key;
+            //IsAttachmentTombstone = true;
+            Collection = "__undefined";
```

Comment:
?

Reply:
this is the Collection name value that was previously used as well